lib/storage/acl.js

```diff
@@ -295,13 +418,55 @@ Acl.prototype.update = function(options, callback) {
 };
 
 /**
+ * Make a {module:storage/bucket} or {module:storage/file} object private.
+ *
+ * This is a short-hand method which will remove ACL permissions for the
+ * "allUsers" scope.
+ *
+ * @param {function} callback - The callback function.
+ *
+ * @alias acl.makePrivate
+ *
+ * @example
+ * myBucket.acl.makePrivate(function(err, aclObject) {});
+ * myFile.acl.makePrivate(function(err, aclObject) {});
+ */
+Acl.prototype.makePrivate = function(callback) {
+  this.delete({
+    scope: 'allUsers'
+  }, callback);
```

Comment:
:us: Flagging this for attention. Not sure this is the intended effect making a bucket/object private is meant to have.